### PR TITLE
Update change-nameservers-at-any-domain-registrar.md

### PR DESCRIPTION
**Pull request by msbemba**
Corrected Step 2 in adding Custom records

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2318

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

File: microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md [hunk 1]
```diff
@@ -157,13 +157,13 @@ For example, here are some additional steps that might be required for email and
 
 1. In the admin center, go to the **Settings** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=834818" target="_blank">Domains</a> page.
 
-3. On the Domains page, select a domain.
+2. On the Domains page, Click on the domain and click on DNS Records tab.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/get-help-with-domains/change-nameservers-at-any-domain-registrar.md]:
```suggestion
2. On the **Domains** page, select the domain and then choose **DNS Records**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@pebaum Please merge this PR. Thanks

**Pull request merged**